config/dlss_baseline.yml

```diff
@@ -84,15 +84,27 @@ Style/BlockComments:
 Style/ClassAndModuleChildren:
   Enabled: false
 
+Style/ClosingParenthesisIndentation:
+  Enabled: false
+
 Style/Documentation:
   Enabled: false
 
+Style/EmptyLinesAroundBlockBody:
+  Enabled: false
+
 Style/EmptyLinesAroundClassBody:
   Enabled: false
 
 Style/EmptyLinesAroundModuleBody:
   Enabled: false
 
+Style/ExtraSpacing:
+  Enabled: false
```

Comment:
I hear you.   The rubocop default is AllowForAlignment set to true, so we agree about that:  https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L425

I'm trying to think about which whitespace affects comprehension and which doesn't;  I could live with leaving this rule on. 

Maybe this is a better example:

```ruby
    @druid = 'druid:aa123bb4567'
    @workspace_root_pathname = Pathname(Dor::Config.cleanup.local_workspace_root)
    @workitem_pathname       = Pathname(DruidTools::Druid.new(@druid, @workspace_root_pathname.to_s).path)
    @workitem_pathname.rmtree if @workitem_pathname.exist?
    @export_pathname         = Pathname(Dor::Config.cleanup.local_export_home)
    @export_pathname.rmtree if @export_pathname.exist?
    @bag_pathname            = @export_pathname.join(@druid.split(':').last)
    @tarfile_pathname        = @export_pathname.join(@bag_pathname + '.tar')
```

doesn't pass:
```ruby
spec/dor/cleanup_service_spec.rb:29:21: C: Unnecessary spacing detected.
    @export_pathname         = Pathname(Dor::Config.cleanup.local_export_home)
                   ^^^^^^^^
```

but I can certainly live with Enabled = true.